ui/app/components/auth/form/oidc-jwt.hbs

```diff
@@ -0,0 +1,48 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<form
+  {{did-insert this.initializeFormData}}
+  {{! OIDC and JWT share the same form and so @authType is the only indication a user has switched tabs.
+  Selecting a different type means we need to re-initialize form data and re-request the role }}
+  {{did-update this.initializeFormData @authType}}
+  {{! manage input changes at the <form> level since "path" and "namespace" are yielded }}
+  {{on "input" this.updateFormData}}
```

Comment:
I know this isn't terribly ideal but since these items are no longer passed in as component args, this seemed like the most reasonable way to track input data